frontend/locales/es.ts

```diff
@@ -0,0 +1,868 @@
+            },
+            error: {
+                invalidFile: 'El tipo de archivo {0} es inválido',
+                noFile: 'No has enviado un archivo',
+                members: {
+                    invalidUser: '{0} no es un usuario válido',
+                    alreadyInvited: '{0} ya está invitado al proyecto',
+                    notMember: '{0} no es un miembro del proyecto y por ello no puedes editar su rol',
+                    invalidRole: '{0} no puede ser añadido/eliminado del proyecto',
+                },
+            },
+            success: {
+                changedIcon: 'Se ha cambiado el icono del proyecto exitosamente',
+                resetIcon: 'Se ha reseteado el icono del proyecto exitosamente',
+                rename: 'Se ha cambiado el nombre del proyecto a {0} exitosamente',
+                softDelete: 'Has borrado este proyecto',
+                hardDelete: 'Has borrado este proyecto totalmente',
+            },
+            tabs: {
+                general: 'General',
+                optional: 'Opcional',
+                management: 'Administración',
+                donation: 'Donación',
+            },
+        },
+        discuss: {
+            login: 'Inicia sesion',
+            toReply: 'para responder a este conversación',
+            noTopic: 'No hay conversación en este proyecto',
+            send: '¡Respuesta publicada!',
+        },
+    },
+    page: {
+        plural: 'Páginas',
+        new: {
+            title: 'Crea una nueva página',
+            error: {
+                minLength: 'Los contenidos de la página son demasiado cortos',
+                maxLength: 'Los contenidos de la página son demasiado largos',
+                duplicateName: 'Ya hay una página con ese nombre',
+                invalidName: 'Nombre inválido',
+                name: {
+                    maxLength: 'El nombre de la página es demasiado largo',
+                    minLength: 'El nombre de la página es demasiado corto',
+                    invalidChars: 'El nombre de la página contiene caracteres inválidos',
+                },
+                save: 'No se ha podido guardar la página',
+            },
+            name: 'Nombre de la página',
+            parent: 'Página Superior (opcional)', //TODO: to be checked
+        },
+        delete: {
+            title: '¿Eliminar página?',
+            text: '¿Seguro que quieres eliminar esta página? Esta operación no se puede deshacer.',
+        },
+    },
+    version: {
+        new: {
+            title: 'Crear versión...',
+            upload: 'Subir archivo',
+            uploadNew: 'Subir una nueva versión',
+            url: 'Escribe una URL',
+            form: {
+                versionString: 'Versión',
+                fileName: 'Nombre del archivo',
+                fileSize: 'Tamaño del archivo',
+                externalUrl: 'URL Externa',
+                hangarProject: 'Proyecto Hangar', //TODO: To be checked
```

Comment:
This one is probably fine.

Reply:
That’s what I thought but I added it to have a second opinion , so I think I’ll remove the “to be checked” comment